File: python/ch01/fibonacci/fibonacci/fibonacci_1.py
def fibonacci(n: int) -> int:
  return fibonacci(n - 1) + fibonacci(n - 2)


def run():
  print("***** Algorithm: Recursive Attempt *****")

  print(f"fibonacci(0) = {fibonacci(0)}")
  print(f"fibonacci(1) = {fibonacci(1)}")
  print(f"fibonacci(5) = {fibonacci(5)}")

File: rust/ch01/fibonacci/src/fibonacci_1.rs
pub fn fibonacci(n: usize) -> usize {
  fibonacci(n - 1) + fibonacci(n - 2)
}

pub fn run() {
  println!("***** Algorithm: Recursive Attempt *****");

  println!("fibonacci(0) = {}", fibonacci(0));
  println!("fibonacci(1) = {}", fibonacci(1));
  // println!("fibonacci(5) = {}", fibonacci(5));
}
